analytics_data_api/v0/serializers.py

```diff
@@ -76,6 +76,29 @@ class Meta(object):
         )
 
 
+class ConsolidatedAnswerDistributionSerializer(ProblemResponseAnswerDistributionSerializer):
```

Comment:
`ProblemResponseAnswerDistributionSerializer` can be removed since it is no longer in use.